test/rqg/datasets/star_schema.sql

```diff
@@ -48,7 +48,7 @@ INSERT INTO d1 VALUES
   (2, 3, 0), (3, 2, 1),
   (3, 4, 0),
   -- 4 has no rows in d1
-  (5, 6, 0),
+  (5, 6, 0), (6, 5, 1),
```

Comment:
Adding rows with inverted `(pk1, pk2)` values to some `dx` tables to produce more results in left joins of the form

```
LEFT JOIN dim_table AS dx ON( ft.fk1 = dx.pk2 AND ft.fk2 = dx.pk1 )
```

or

```
LEFT JOIN dim_table AS dx ON( dy.pk1 = dx.pk2 AND dy.pk2 = dx.pk1 )
```